WMCDuplicateRemover/Code/Serialization/ObjectDeserializer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace WMCDuplicateRemover.Code.Serialization
+{
+    public class ObjectDeserializer
+    {
+        public T ReadFromBinaryFile<T>(string filePath)
+        {
+            try
+            {
+                using (var stream = File.Open(filePath, FileMode.Open))
```

Comment:
[File.Open(string, FileMode)](https://msdn.microsoft.com/en-us/library/b9skfh7s(v=vs.110).aspx) returns a [FileStream](https://msdn.microsoft.com/en-us/library/system.io.filestream(v=vs.110).aspx) type.

I typically use var everywhere I can, its a personal coding style choice. 

var is nice because I don't need to know what type is on the right hand side of the assignment, the compiler will figure that out for me. 
It can also be incentive to use good variable names and not rely on the type identifier.

I've seen people decide to name something so short because the type is so long:
`ReallyCoolClassThatDoesX x = new ReallyCoolClassThatDoesX();`

Anyway, it's a personal choice and I'm sure you can find a lot of info for and against using var. I'm sure the 'right' answer is somewhere in the middle. [Here's an article advocating something in between](https://blogs.msdn.microsoft.com/ericlippert/2011/04/20/uses-and-misuses-of-implicit-typing/)